src/nvim/memline.c

```diff
@@ -167,17 +167,15 @@ enum {
   B0_MAGIC_CHAR = 0x55,
 };
 
-// Block zero holds all info about the swap file. This is the first block in
-// the file.
+// Block zero holds all info about the swapfile. This is the first block in the file.
 //
-// NOTE: DEFINITION OF BLOCK 0 SHOULD NOT CHANGE! It would make all existing
-// swap files unusable!
+// NOTE: DEFINITION OF BLOCK 0 SHOULD NOT CHANGE! It would make all existing swapfiles unusable!
 //
 // If size of block0 changes anyway, adjust MIN_SWAP_PAGE_SIZE in vim.h!!
 //
 // This block is built up of single bytes, to make it portable across
 // different machines. b0_magic_* is used to check the byte order and size of
-// variables, because the rest of the swap file is not portable.
+// variables, because the rest of the swapfile is not portable.
```

Comment:
Haha do you also have very strong opinions about `swapfile` versus `swap file` @justinmk 😆?

Reply:
:) not really, but trying to be consistent at least internally. "swap file name" reads awkwardly. 

I could be persuaded to revert this though.

Reply:
I don't have strong feelings about it, but I agree that "swapfile" reads better.